pkgs/tools/text/kdiff3/default.nix

```diff
@@ -1,35 +1,18 @@
 {
-  mkDerivation, lib, fetchgit, fetchpatch,
+  mkDerivation, lib, fetchurl, # fetchgit, fetchpatch,
```

Comment:
Is there value to leaving in commented-out parameters?

Reply:
Oops, this was just a mistake :innocent:.  Will fix shortly.